content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -58,6 +58,12 @@ equal `PodFailurePolicy`. Additionally, the `message` field contains a
 more detailed information about the Job termination, such as:
 `Container main for pod default/job-pod-failure-policy-failjob-8ckj8 failed with exit code 42 matching FailJob rule at index 0`.
 
+{{< note >}}
+You will also observe the `FailureTarget` Job condition, with the same `reason`
+and `message` as for the the `Failed` condition (see also
+[Job termination and cleanup](#job-termination-and-cleanup)).
+{{< /note >}}
+
```

Comment:
Try this:

In the Job status, the following conditions display:

- `Failed` condition: has a `reason` field set to `PodFailurePolicy` and a
   `message` field with more information about the termination, like
   `Container main for pod default/job-pod-failure-policy-failjob-8ckj8 failed with exit code 42 matching FailJob rule at index 0`.
   The Job controller adds this condition after all of the Job's Pods are terminated.
- `FailureTarget` condition: same reason and message as the `Failed` condition. The Job
   controller adds this condition as soon as the Job is considered a failure. For details, see
   [Termination of Job Pods](/docs/concepts/workloads/controllers/job/#termination-of-job-pods).

Reply:
Makes sense, updated along the lines, but first mention `FailureTarget` and then `Failed` to reflect the natural order of them being added.